specification/crds/go/pkg/apis/serverlessworkflow/v1alpha2/zz_generated.openapi.go

```diff
@@ -0,0 +1,15 @@
+// +build !ignore_autogenerated
```

Comment:
same question about generated code here.

Reply:
This one generates the boilerplate code for the open api validation, again useful for clients. But this can go away.

Reply:
already answered in previous comment.